### PR TITLE
Added config to make add to queue responses for requester only

### PR DESCRIPTION
**Pull request by Sheeley7**
Added config to make add to queue responses for requester only

This was my first time using Prisma, so you may want to verify that database migration is working.

### CODE REVIEW

**Review by codetheweb — APPROVED** [review]
thank you!

File: src/commands/config.ts [hunk 1]
```diff
@@ -33,6 +33,13 @@ export default class implements Command {
         .setName('value')
         .setDescription('whether to leave when everyone else leaves')
         .setRequired(true)))
+    .addSubcommand(subcommand => subcommand
+      .setName('set-queue-add-response-eph')
+      .setDescription('set whether bot responses to queue adds are only displayed to the requester')
+      .addBooleanOption(option => option
+        .setName('value')
+        .setDescription('whether bot responses to queue adds are only displayed to the requester')
```
Comment (codetheweb) [inline src/commands/config.ts:41]:
```suggestion
    .addSubcommand(subcommand => subcommand
      .setName('set-queue-add-response-hidden')
      .setDescription('set whether bot responses to queue additions are only displayed to the requester')
      .addBooleanOption(option => option
        .setName('value')
        .setDescription('whether bot responses to queue additions are only displayed to the requester')
```

**Review by codetheweb — COMMENTED** [review]
made some wording changes but didn't have permission to push to your brach

**Comment by Sheeley7** [comment]
@codetheweb is anything else needed to get this merged into the main branch?

**Comment by codetheweb** [comment]
nope, just been busy lately :)

lgtm

**Pull request merged**